src/components/button.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = styled.a`
+  background: ${props => props.light ? "white" : "#202020"};
+  color: ${props => props.light ? "#202020" : "white"};
+  display: block;
+  font-size: 1.5rem;
+  height: 4rem;
+  letter-spacing: 0.05em;
+  line-height: 4rem;
+  margin: ${props => props.noMargin ? "0" : "5rem auto 3rem"};
+  max-width: 21rem;
+  min-width: 10rem;
+  text-align: center;
+  text-transform: uppercase;
```

Comment:
I realize we are using the button component for the menu items, as well. Since those need to be all uppercase, maybe we make it a configurable style? The buttons I'm referring to that don't need capitalization are `Documentation`, `Copy`, `View All`, `Contact`, and `Careers`

Reply:
Same as above